assets/training/finetune_acft_hf_nlp/components_v2/finetune/text_generation/spec.yaml

```diff
@@ -15,7 +15,7 @@ code: ../../../src/
 inputs:
   mlflow_model_path:
     type: mlflow_model
-    optional: True
+    optional: False
```

Comment:
do we need mlflow model to be required? should we allow pytorch model input too?